internal/constants/constants.go

```diff
@@ -164,6 +164,9 @@ const DateTimeFormatRecord = "Mon Jan 2 2006 15:04:05 -0700 MST"
 // PlatformSignupURL is the account creation url used by the platform
 const PlatformSignupURL = "https://platform.activestate.com" + "/create-account"
 
+// TrayDocumentationURL is the url for the state tool documentation to be used in the state tray application
+const TrayDocumentationURL = "http://docs.activestate.com/platform/state/?utm_source=platform-application-gui&utm_medium=activestate-desktop&utm_content=drop-down&utm_campaign=maru"
```

Comment:
This doesn't look right, it just points to the general state tool docs.

Reply:
What should it be pointing to?

Reply:
Oh I see, this just points to the general documentation, but FROM the systray. That confused me, I thought this was the documentation FOR the systray.